### PR TITLE
fiodriver: fix sending of frame 62 for CMU fail state action

### PR DESCRIPTION
**Pull request by gallagjm**
Allow CMU frame 62 to be scheduled with fio_fiod_frame_schedule_set().
Fix frame 62 payload bits to correctly reflect fio_cmu_fsa enumeration.
Fix payload handling for fio_fiod_frame_write().

### CODE REVIEW

File: fio/src/fiodriver/fioman.c [hunk 1]
```diff
@@ -726,13 +726,9 @@ fioman_add_def_fiod_frames
...
 				list_add_tail( &((FIOMSG_RX_FRAME *)(rx_frame))->elem, rx_frames );
 
-#if 0
-			/* Indicate other valid frames not sent by default (FIO332,FIOTS1,FIOINSIU,FIOOUTSIU) */
-			p_fiod->frame_frequency_table[60] = FIO_HZ_0;
-			p_fiod->frame_frequency_table[61] = FIO_HZ_0;
+			/* Indicate other valid frames not sent by default (FIOCMU) */
 			p_fiod->frame_frequency_table[62] = FIO_HZ_0;
-			p_fiod->frame_frequency_table[65] = FIO_HZ_0;
-#endif
```
Comment (dtarico) [inline fio/src/fiodriver/fioman.c:735]: Shouldn't we allow 60, 61, 65?

Comment (gallagjm) [reply fio/src/fiodriver/fioman.c:735]: So, in ATC5401, frames 60 and 65 are "Application program driven. Not scheduled.".
Frame 61 is the long status version of default frame 67, which we have never used.
Both frames 61 and 65 are deprecated in ATC5301.

Comment (dtarico) [reply fio/src/fiodriver/fioman.c:735]: Good point about 60 and 65. Don't think there would be any need to schedule them. I think we should allow 61, for use with ITS cabinets.

Comment (gallagjm) [reply fio/src/fiodriver/fioman.c:735]: We will need to add support for frame 61 as well as allowing it to be scheduled. Would be better in a separate commit?

Comment (dtarico) [reply fio/src/fiodriver/fioman.c:735]: Oh, then let's not bother with frame 61 for now. I was thinking that the support was already there.

**Review by dtarico — APPROVED** [review]

**Pull request merged**